MuxExoPlayer/build.gradle

```diff
@@ -97,6 +97,13 @@ android {
                             into "${buildDir}/outputs/artifacts/"
                             rename("${project.name}-${flavor.name}-release.aar", "mux-${versionName}.aar")
                             rename("mapping.txt", "mapping-${versionName}.txt")
+                            File aarFile = new File("${buildDir}/outputs/artifacts/mux-${versionName}.aar")
+                            println "===++++ >>>>> ${aarFile.name} size = ${aarFile.size()}, exists ${aarFile.exists()}";
+                            if ( aarFile.length() > 120000 ) {
```

Comment:
Does `aarFile.length()` return bytes? If so, this is comparing against 120,000 bytes, right?

Reply:
Yes it does, it will print the file sizes on stdouit if you run build task, you will see each file size.